contentprovider.go

```diff
@@ -149,8 +175,14 @@ func (p *contentProvider) fillMatches(ms []*candidateMatch) []LineMatch {
 
 			result = []LineMatch{res}
 		}
-	} else {
-		ms = breakMatchesOnNewlines(ms, p.data(false))
+
+	default:
+		if ms[0].scope == query.ScopeSymbol {
```

Comment:
nit: I think it is clearer to not use default, and rather handle query.ScopeSymbol separately and duplicate the two lines below.